enqueue/fs/0.4/manifest.json

```diff
@@ -0,0 +1,5 @@
+{
+    "env": {
+        "ENQUEUE_DSN": "file:/%VAR_DIR%/enqueue"
```

Comment:
It's a problem. On Unix system, using `file://` will result in paths like `file:///var/www/dev/enqueue` which might not work, whereas on windows it will result in `file://e:/www/dev/enqueue` which should work.

Dunno how to fix this 😱